.travis.yml

```diff
@@ -10,7 +10,7 @@ env:
     - LD_LIBRARY_PATH=/tmp/protobuf/lib
 
 before_install:
-  - rm ~/.m2/settings.xml || true # Avoid repository.apache.org, which has QPS limits and is useless
+  - cp .travis.settings.xml ~/.m2/settings.xml
```

Comment:
This still deserves some comments. It is important, for example, that this overwrites the default config which references repository.apache.org.